### ISSUE TITLE
Write a docker.bat script for Windows

### ISSUE THREAD

**Issue by krushiraj** [coderplex/learnplex-backend#4]
Right now we have `docker.sh` script to handle docker related things. Write an equivalent script for Windows CMD.

**Comment by abhishekgpta** [comment]
don't need docker bat script ran docker.sh using git bash shell in windows @krushiraj

**Comment by krushiraj** [comment]
Yeah that is a solution. I just raised this issue if we might need a pure windows solution. People can even used bash in windows cmd which comes with the lastest builds of windows 10

To make sure, can you please check if the current script works fine in git bash if you would like to work on this issue.

**Comment by abhishekgpta** [comment]
I did check, I am using windows 10
